frontend/src/components/Post.js

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import clsx from 'clsx';
+import Card from '@material-ui/core/Card';
+import CardHeader from '@material-ui/core/CardHeader';
+import CardMedia from '@material-ui/core/CardMedia';
+import CardContent from '@material-ui/core/CardContent';
+import CardActions from '@material-ui/core/CardActions';
+import Collapse from '@material-ui/core/Collapse';
+import Avatar from '@material-ui/core/Avatar';
+import IconButton from '@material-ui/core/IconButton';
+import Typography from '@material-ui/core/Typography';
+import { red } from '@material-ui/core/colors';
+import FavoriteIcon from '@material-ui/icons/Favorite';
+import ShareIcon from '@material-ui/icons/Share';
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
+import MoreVertIcon from '@material-ui/icons/MoreVert';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    maxWidth: 345,
+  },
+  media: {
+    height: 0,
+    paddingTop: '56.25%', // 16:9
+  },
+  expand: {
+    transform: 'rotate(0deg)',
+    marginLeft: 'auto',
+    transition: theme.transitions.create('transform', {
+      duration: theme.transitions.duration.shortest,
+    }),
+  },
+  expandOpen: {
+    transform: 'rotate(180deg)',
+  },
+  avatar: {
+    backgroundColor: red[500],
+  },
+}));
+
+//pass properties to ActivitiesPost function
+
+const ActivitiesPost = (props) => {
```

Comment:
With our linting rules, all components need to use [prop destructuring](https://medium.com/@lcriswell/destructuring-props-in-react-b1c295005ce0). This makes it really easy to see what properties a component uses and makes code more readable.

Along with this, our linter also requires using [prop-types](https://www.npmjs.com/package/prop-types) to validate the types of the properties being passed to our components. This requires including `prop-types` and then defining the validation object before exporting the component. Refer to [the usage](https://www.npmjs.com/package/prop-types#usage) for help on this.